scripting/include/get5.inc

```diff
@@ -355,20 +355,18 @@ methodmap Get5Player < JSON_Object {
       }
     }
 
-    public bool IsEqualToPlayerId(const char[] id) {
-      char ownId[64];
-      this.GetSteamId(ownId, sizeof(ownId));
-      return StrEqual(ownId, id);
-    }
-
-    public bool IsEqualToPlayer(const Get5Player player) {
-      char playerId[64];
-      player.GetSteamId(playerId, sizeof(playerId));
-      return this.IsEqualToPlayerId(playerId);
+    property int Id {
+      public get() {
+        return this.GetInt("id");
+      }
+      public set(int id) {
+        this.SetInt("id", id);
+      }
```

Comment:
I would suggest making this explicitly called `UserId`, or at least adding a comment what kind of "id" this is.